backend/src/main/java/de/amos/apachepulsarui/service/ClusterService.java

```diff
@@ -85,4 +89,20 @@ private List<String> getTenantsAllowedForCluster(String clusterName) throws Puls
                 .toList();
     }
 
+    private ClusterDto enrichWithCardDetails(ClusterDto clusterDto) {
+        List<String> tenats = getTenantsAllowedForCluster(clusterDto.getName());
+        long numberOfNamespaces = tenats.stream().mapToLong(this::getNumberOfNamespacesForTenant).sum();
+        clusterDto.setNumberOfTenants(tenats.size());
+        clusterDto.setNumberOfNamespces(numberOfNamespaces);
+        return clusterDto;
+    }
+
+    private long getNumberOfNamespacesForTenant(String tenant) {
+        try {
+            return pulsarAdmin.namespaces().getNamespaces(tenant).size();
```

Comment:
Can't we use the method from the namespace service?
`namespaceService.getAllOfTenant(tenant).size()`